app/src/main/java/com/example/drawingApp/MainViewModel.kt

```diff
@@ -10,4 +11,12 @@ class MainViewModel : ViewModel() {
     @ColorInt
     var primaryColor = Color.BLACK
 
+    val layerNames: MutableList<String> = mutableListOf()
+
+    var activeBitmap: Bitmap? = null
+
+    var isDialogOpen = false
+    var currentDialogText = ""
```

Comment:
Which dialog for these? Name more clear

Reply:
updated to isLayerCreationDialogOpen, and currentLayerCreationDialogText